### DISCUSSION TITLE
no ads being detected or clicked

### DISCUSSION THREAD

**Discussion by GameDevKitY** [dhowe/AdNauseam#2529] [Q&A]
It blocks them just fine but it doesnt click on them and i verifed there is ads on the site by turning it off and seeing them and turning back on and not seeing them
its also doesnt click on other sites either
other exenstions I have are Honey, Lastpass, Mullvad brownser extension, Ruffle flash emu, sponsorblock, startpage, twp (translate web pages)
AdNauseam: 3.21.0
Firefox: 124
filterset (summary):
 network: 131633
 cosmetic: 46341
 scriptlet: 19051
 html: 1767
listset (total-discarded, last-updated):
 default:
  user-filters: 4-0, never
  adnauseam-filters: 538-2, 3h.16m
  eff-dnt-whitelist: 589-0, 3h.16m
  ublock-filters: 37656-166, 57m Δ
  ublock-badware: 8183-0, 57m Δ
  ublock-privacy: 793-0, 57m Δ
  ublock-unbreak: 2305-2, 57m Δ
  ublock-quick-fixes: 140-0, 57m Δ
  easylist: 85982-200, 57m Δ
  easyprivacy: 50766-23, 57m Δ
  urlhaus-1: 12315-5, 3h.16m
filterset (user): [array of 4 redacted]
userSettings:
 admap: [redacted]
 firstInstall: false
 hidingAds: true
 clickingAds: true
 blockingMalware: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 261 ms (selfie)
 maxAssetCacheWait: 105 ms
 cacheBackend: indexedDB
 |
 
 AdNauseam: 3.21.0
Firefox: 124
filterset (summary):
 network: 131633
 cosmetic: 46341
 scriptlet: 19051
 html: 1767
listset (total-discarded, last-updated):
 default:
  user-filters: 4-0, never
  adnauseam-filters: 538-2, 3h.16m
  eff-dnt-whitelist: 589-0, 3h.16m
  ublock-filters: 37656-166, 57m Δ
  ublock-badware: 8183-0, 57m Δ
  ublock-privacy: 793-0, 57m Δ
  ublock-unbreak: 2305-2, 57m Δ
  ublock-quick-fixes: 140-0, 57m Δ
  easylist: 85982-200, 57m Δ
  easyprivacy: 50766-23, 57m Δ
  urlhaus-1: 12315-5, 3h.16m
filterset (user): [array of 4 redacted]
userSettings:
 admap: [redacted]
 firstInstall: false
 hidingAds: true
 clickingAds: true
 blockingMalware: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 261 ms (selfie)
 maxAssetCacheWait: 105 ms
 cacheBackend: indexedDB

**Comment by GameDevKitY** [comment]
and on the little adblocker icon in the drop down menu it says

Blocked since install549 (1%)

but the actual adnesum isnt working

**Comment by dhowe** [comment]
@mneunomne ideas here? do we still have a simple test page ?

**Comment by vanyasem** [comment]
The uBlock portion of the extension works fine for me, but I was unable to get even a single Ad added to the AdNauseum vault. Brave Browser on Windows, installed through a whitelisted `crx`.
I'm in Russia.

**Comment by dhowe** [comment]
perhaps create an issue for this?

**Reply by vanyasem** [reply]
I commented on https://github.com/dhowe/AdNauseam/issues/2585, which seems to be the meta issue for such complaints
Do you want me to open a separate issue?
